pkg/operator/ceph/provisioner/provisioner.go

```diff
@@ -48,6 +68,16 @@ type RookVolumeProvisioner struct {
 	flexDriverVendor string
 }
 
+type cephFSProvisioner struct {
```

Comment:
What about having a different code file for cephfs provisioner in the same package?

Reply:
> What about having a different code file for cephfs provisioner in the same package?

i think it's unnecessary, because cephfs provisioner or rbd(block) provisioner are belongs to storage provisioner, it's reasonable in the same provisioner.go file.

hope your replay @rohan47